src/tests/Microsoft.PowerFx.Core.Tests/TypeSystemTests/DTypeTests.cs

```diff
@@ -205,7 +208,36 @@ public void UnknownIsSubtypeOfAll()
             Assert.True(MultiSelectOptionSetType.Accepts(DType.Unknown));
             Assert.True(DType.Polymorphic.Accepts(DType.Unknown));
         }
-        
+
+        [Fact]
+        public void DeferredIsSubtypeOfAll()
```

Comment:
Also good to test other type operations:
Supertype, Union, CoercesTo, ...

and operations where deferred is a field of an aggregate (table/record)

Reply:
Since DeferredType is SubType of all and we have a test for that. SuperType, Union would be redundant. DefeeredType is not usable pass Type checking so there would be no need to check CoercesTo either...

Reply:
Since DeferredType is SubType of all and we have a test for that. SuperType, Union would be redundant. DefeeredType is not usable pass Type checking so there would be no need to check CoercesTo either...

Reply:
specifically add tests that cover that, then